src/common/store.ts

```diff
@@ -1,10 +1,13 @@
 import _ from 'lodash/fp'
+import {Observable} from 'rxjs'
 import {set as mutatingSet} from 'lodash'
 import ElectronStore from 'electron-store'
 import {gt} from 'semver'
 import {config} from '../config/renderer'
 import {DATADIR_VERSION} from '../shared/version'
 
+type OnDidChangeData<T> = {oldVal: T; newVal: T}
```

Comment:
Why not just `DidChangeData` or `Change`?

Reply:
This is the data from `onDidChange`, so `onDidChangeData` is appropriate IMHO.